src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1545,6 +1545,39 @@ public static IEnumerable<IEdmOperationImport> OperationImports(this IEdmEntityC
             return container.AllElements().OfType<IEdmOperationImport>();
         }
 
+        /// <summary>
+        /// Returns NavigationPropertyBindings with their corresponding container elements (EntitySet or Singelton)
+        /// </summary>
+        /// <param name="container">Reference to the calling object.</param>
+        /// <returns>collection of pairs of container element and NavigationPropertyBindings.</returns>
+        public static IEnumerable<(IEdmEntityContainerElement, IEdmNavigationPropertyBinding)> GetNavigationPropertyBindings(this IEdmEntityContainer container)

```

Comment:
```suggestion
        public static IEnumerable<Tuple<IEdmEntityContainerElement, IEdmNavigationPropertyBinding>> GetNavigationPropertyBindings(this IEdmEntityContainer container)
```
Concise syntax not supported in C# 7.0 - that's why the tests are failing